### ISSUE TITLE
3D human pose estimation in video with temporal convolutions and semi-supervised training

### ISSUE THREAD

**Issue by yj-song131** [GDSC-Ewha-4th/Study-paperReview#83]
* abstract
  + 이 논문에서는 비디오 환경에서 2차원 keypoint들에 대해 diated temporal convolution 기반의 fully convolutional model을 적용해 3차원 pose estimation을 수행한다. 
  + 또한 unlabeled video data를 활용하는 간단하고 효율적인 semi-supervised 학습 방법인 back-projection을 소개한다. 
  + Supervised setting에서 이 논문은 Human3.6M 데이터에서 11% 정도의 error reduction을 달성하였다.
* Model
![image](https://user-images.githubusercontent.com/97579755/210213605-4bb85be9-b835-4c93-af2e-df4f293b49c6.png)

  + 이 모델은 2차원 keypoint sequences를 입력으로 받아 temporal convolution을 통해 변환하는 residual connection이 포함된 fully convolutional 구조이다. 
  + CNN 모델은 parallelization을 가능하게 하지만 RNN은 시간에 따른 parallelization을 할 수 없다. Convolution Model에서 path of the gradient는 고정된 길이를 가지기 때문에, Vanishing 문제 및 exploding gradient 문제를 완화시킬 수 있다.
  + input layer는 각 joint J의 연결된 (x,y) 좌표를 취하게 되고, 커널 크기인 W 및 출력 채널인 C와 함께 temporal convolution을 적용한다. 그 다음 skip-connection으로 연결된 B(ResNet-style black)이 이어진다. 각 블록은 먼저 W 및 dilation factor D = W^B로 1D conv를 수행한 다음 커널 크기 1로 convolution을 수행하게 된다. 
  + Convolution 이후에는 Batch Normalization, ReLU, Dropout이 이어진다. 마지막 layer는 시간 정보를 활용하기 위해 과거 및 미래의 모든 데이터를 사용해 sequence의 모든 frame에 대한 pose estimation을 출력한다.
* Semi-Supervised Approach
![image](https://user-images.githubusercontent.com/97579755/210213711-96cbb544-439c-4234-990c-aff39204ed82.png)
  + 이 논문에서는 공개되어 있는 video를 2차원 keypoint 탐지기와 결합하여 back-projection loss term으로 supervised loss function을 확장한다. 
  + 따라서 unlabeled data에 대한 auto-encoding 문제를 해결하고 encoder는 pose estimator, decoder는 projection layer의 역할을 수행한다. 
  + 만약 decoder의 2차원 joint 좌표가 원래 입력에서 멀리 떨어져 있는 경우 페널티를 부여받는다.
  + 이 파트에서는 총 두 가지 목표가 있는데, labeled data가 batch의 first-half를 차지하고 unlabeled data가 batch의 나머지를 차지하도록 최적화 시킨다. 
  + Labeled data의 경우 GT 3차원 pose를 사용하고 supervised loss를 학습한다. Unlabeled data는 예측된 3차원 pose가 2차원으로 다시 투영된 다음 autoencoder의 Loss를 구현하는 데 사용한다.

* Result
![image](https://user-images.githubusercontent.com/97579755/210213877-d9673508-b0c6-498f-8921-f0969bb60e3f.png)
